kong/plugins/zipkin/handler.lua

```diff
@@ -25,7 +25,7 @@ local function log(premature, reporter, name)
 
 	local ok, err = reporter:flush()
 	if not ok then
-		ngx.log(ngx.ERR, "[", name, "] reporter flush ", err)
+		kong.log.err("[", name, "] reporter flush ", err)
```

Comment:
Ditto.

Reply:
I think this is also needed because the `name` here is the name of an reporter, not the plugin name.

Reply:
The reporter name is passed in from the plugin name. Please go ahead and remove the `[name]`.